server/app/app.go

```diff
@@ -173,6 +174,9 @@ func (a *App) Start() error {
 		return err
 	}
 
+	// Start otlp endpoint
+	go func() { otlp.StartServer(21321, a.db) }()
```

Comment:
This is a goroutine right?

Reply:
Yes it is. As this is a blocking operation, we need to start it in another thread to be able to run the HTTP server and this server at the same time